VideoApi/VideoApi.DAL/packages.lock.json

```diff
@@ -1939,11 +1939,18 @@
           "VideoApi.Domain": "1.0.0"
         }
       },
+      "videoapi.contract": {
+        "type": "Project",
+        "dependencies": {
+          "System.ComponentModel.Annotations": "4.7.0"
+        }
+      },
       "videoapi.domain": {
         "type": "Project",
         "dependencies": {
           "Enums.NET": "4.0.0",
-          "System.ComponentModel.Annotations": "4.7.0"
+          "System.ComponentModel.Annotations": "4.7.0",
+          "VideoApi.Contract": "1.0.0"
```

Comment:
after moving the dependency of the contract from the domain layer please ensure we remove this contract reference from DAL,